### DISCUSSION TITLE
Fonts render different when compared to iTerm

### DISCUSSION THREAD

**Discussion by paulopacitti** [ghostty-org/ghostty#4178] [Q&A] [answered]
I'm trying to migrate from iTerm to Ghostty. I don't use any special features, so transitioning to Ghostty would be fine. But the fonts are not rendering the same.

Ghostty get the fonts more shorter and with lower resolution. Here's the comparison between the same fonts sizes of iTerm and Ghostty:
- iTerm: 
![iTerm](https://github.com/user-attachments/assets/de188dbd-0f87-4241-b6cc-8649f1fdca9e)
- Ghostty: 
![Ghostty](https://github.com/user-attachments/assets/8bbf34c3-f550-48a5-a816-e748dc0e37fb)

Here's my current config file:
```
font-family = "JetBrainsMonoNL Nerd Font Mono"
font-size = 13
freetype-load-flags = no-force-autohint
bold-is-bright = true
theme = "GruvBoxDarkHard"
background-opacity = 0.8
background-blur-radius = 20

adjust-cell-height = -1

window-padding-x = 4
window-padding-y = 2

confirm-close-surface = false
```

**Is there any missing configuration that I have to do? Or it's actually bug?** Thanks in advance :)

**Comment by rhodes-b** [comment]
https://ghostty.org/docs/config/reference#font-thicken

**Reply by paulopacitti** [reply]
This does not solve the issue, I've already tried that :/ Example with `font-thicken`:

- iTerm: 
![iTerm](https://github.com/user-attachments/assets/46c9ccba-ce84-4d90-a420-3df2674af942)

- Ghostty: 
![Ghostty](https://github.com/user-attachments/assets/0460ff95-54e1-46d4-8c2a-359609eb9c2c)

**Reply by rhodes-b** [reply]
It looks exactly the same to me besides the cell size 
which you set
`adjust-cell-height = -1` I assume that was for the neovim bottom line thing?

you could also try increasing `font-size`

**Reply by mattgreen** [reply]
For me, the difference was the cell height. Setting `adjust-cell-height = -2` was sufficient to get it extremely close to both iTerm 2 and Terminal.app. I hesitate to say this should be default, but I suspect other people are going to ask this question as well, and I think the more condensed cell height looks better overall.

Edit: Also had to set `font-thicken=true` to get the correct weight on the same font compared to iTerm.

**Reply by paulopacitti** [reply]
> you could also try increasing font-size

that's the thing, I want this font-size (12), as the same in iTerm

> Edit: Also had to set font-thicken=true to get the correct weight on the same font compared to iTerm.

as said in my reply, the screenshots are already with `font-thicken=true`. Setting `adjust-cell-height = -2` did not change the overall look, font is still rendered with a short height when compared with iTerm

**Comment by isavcic** [comment]
Just to add, `adjust-cell-width = -1` brings it even closer to iTerm.

**Comment by mitchellh** [comment, marked as answer]
Our font metrics follow the font exactly, and we've done a lot of diligence to ensure they're correct. Still, there may be bugs but it'd be more helpful if you could provide more details about specific glyphs being wrong if that's the case.

Grid size itself is not standardized. It's not specified by the font and most terminals calculate it differently to the point where you'll see subtle differences. We provide the `adjust-` configs for this reason.

**Reply by paulopacitti** [reply]
thanks for the response! Using the `adjust-<>` configs definelty improved the looks to look closer to iTerm's rendering ❤️
